library.json

```diff
@@ -12,8 +12,9 @@
         "flags": [
             "-I Inc",
             "-I OD",
-            "-D LUOS_REV='v0.6'"
+			"-D MYVERSION=\"1.2.3\""
```

Comment:
did you try to remove the "" and put it in the C code around LUOS_REV. > `sizeof("LUOS_REV"))`

I don't know if its working...